.gitattributes

```diff
@@ -1,2 +0,0 @@
-* text=auto eol=lf
-upstream/** linguist-vendored
```

Comment:
We should keep this .gitattributes file, though. `* text=auto eol=lf` in specific ensures Windows contributors can't mess up with line endings so easily. The `upstream/**` part is harmless to have.

I'll still merge, though. I'll just add this back later.

Reply:
Sorry, I missed that first line!